robocat/cadmin/200-rbac.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cadmin
+  namespace: default
```

Comment:
i see we have `cadmin` in `tektoncd` also - this one in `default` doesnt seem to get any bindings, do we need it?

Reply:
im also not completely clear on the difference in the intended usage for` tekton-deployer` vs `cadmin` (i dont see `tekton-deployer` in the docs - maybe it's in some docs that already exist?

Reply:
oh, nice catch, thank you - I moved this to root and forgot to remove it from here
'tekton-deployer' is the admin of 'tekton-pipelines' so it will be used to deploy tekton services. I could have add this in the next PR, sorry